src/core/friendly_errors/fes_core.js

```diff
@@ -168,6 +176,200 @@ if (typeof IS_MINIFIED !== 'undefined') {
     console.log(translator('fes.pre', { message }));
   };
 
+  const editDistance = (w1, w2) => {
```

Comment:
See above comment -- maybe `computeEditDistance` would be better so that 'edit' doesn't read as the verb.

Reply:
Yeah makes sense. I will change the name